config/settings.yml

```diff
@@ -116,3 +116,5 @@ rails_performance:
 
 file_storage:
   default_root_path: <%= ::Rails.root %>/data/<%= ::Rails.env %>/
+
+rmd_convert_enabled: true
```

Comment:
The default should be `false` (you can use `true` for testing purposes, but the code that's merged in should have this set to `false`)